app/scripts/head/startup-styles.js

```diff
@@ -103,11 +103,8 @@
       /**
        * The iframe'd OAuth flow needs special styling applied to it as
        * soon as possible so that it doesn't look terrible.
-       *
-       * Make sure that the iframe name is not 'remote'
-       * which is used by 'about:accounts'.
        */
-      if (this.environment.isFramed() && this.window.name !== 'remote') {
+      if (this.environment.isFramed()) {
```

Comment:
This change concerns me a bit. The purposes of this method is to apply special styling:
```
The iframe'd OAuth flow needs special styling applied
```
but the about:accounts is also an iframed flow? How does it differentiate that now?